### PR TITLE
Fix the logs pattern error

### PR DESCRIPTION
**Pull request by zhougw001**
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by nilo19** [comment]
Thanks @zhougw001, please following the link above to get the CLA Authorization.
/ok-to-test

**Comment by feiskyer** [comment]
@zhougw001 thanks for the contribution. Could you sign CLA?

File: cmd/cloud-controller-manager/app/core.go [hunk 1]
```diff
@@ -213,7 +213,7 @@ func startNodeIpamController(ctx context.Context, completedConfig *cloudcontroll
 		// should be dual stack (from different IPFamilies)
 		dualstackServiceCIDR, err := netutils.IsDualStackCIDRs([]*net.IPNet{serviceCIDR, secondaryServiceCIDR})
 		if err != nil {
-			return nil, false, fmt.Errorf("failed to perform dualstack check on serviceCIDR and secondaryServiceCIDR error:%w", err)
+			return nil, false, fmt.Errorf("failed to perform dualstack check on serviceCIDR and secondaryServiceCIDR error:%v", err)
```
[the remark this answers is not in the document]
Comment (feiskyer) [reply cmd/cloud-controller-manager/app/core.go:216]: Besides the above, could you also squash the commits?

**Comment by MartinForReal** [comment]
@zhougw001 Do you still have bandwidth to deal with these code changes?

**Pull request closed without merging**